targets/targets.json

```diff
@@ -7965,12 +8019,21 @@
             "network-default-interface-type": "WIFI"
         }
     },
+    "CYW943012P6EVB_01_M0": {
+        "inherits": ["MCU_PSOC6_M0"],
+        "supported_toolchains": ["GCC_ARM", "ARM", "IAR"],
```

Comment:
No need to specify supported_toolchains here as the parent MCU_PSOC6 already defines the same toolchain set as here.

Reply:
Now that we have switched to ARMC6, these entries for supported_toolchains must change.
Cypress targets ares still using ARM compiler 5, and should change all "ARM" entries to "ARMC5" in supported_toolchains. Please make this change for all Cypress targets.

Reply:
done